src/components/Footer.js

```diff
@@ -0,0 +1,50 @@
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+import ReturnButton from '../components/ReturnButton.js';
+import dashButton from '../imgicon/dash-button.png';
+
+import ConfigPersonalButton from './ConfigPersonalButton.js';
+
+export default function Footer() {
+  return (
+    <FooterMenu className="StyledFooterMenu">
+      <ReturnButton />
+      <Link className="StyledDashLink" to={'/home'}>
+        <img src={dashButton} alt="home button" />
+      </Link>
+      <ConfigPersonalButton />
+    </FooterMenu>
+  );
+}
+
+const FooterMenu = styled.footer`
+  height: 50px;
+  width: 100%;
+  background-color: black;
+  display: flex;
+  justify-content: space-around;
+  position: absolute;
+  bottom: 0;
+
+  .StyledDashLink {
```

Comment:
It is better to define a separate styled component instead to define a nested class.

Reply:
Ok, I rebuilt it - it works!